plugins/ptp_operator/ptp_operator_plugin.go

```diff
@@ -245,14 +245,16 @@ func getCurrentStatOverrideFn() func(e v2.Event, d *channel.DataChan) error {
 					case ptp.PtpClockClassChange:
 						clockClass := fmt.Sprintf("%s/%s", string(ptpInterface), ptpMetrics.ClockClass)
 						data = processDataFn(data, eventManager.GetPTPEventsData(s.SyncState(), s.ClockClass(), clockClass, eventType))
-					// overallSyncState is only for master
 					case ptp.SyncStateChange:
 						overallSyncState = getOverallState(overallSyncState, s.SyncState())
 					}
 				case ptpMetrics.ClockRealTime:
 					switch eventType {
 					case ptp.OsClockSyncStateChange:
 						data = processDataFn(data, eventManager.GetPTPEventsData(s.SyncState(), s.LastOffset(), string(ptpInterface), eventType))
```

Comment:
check line #472 and #488 when ptpconfig delete is detected , an event is sent out

Reply:
that is covered in line #272-279 in plugins/ptp_operator/metrics/manager.go